CoreTweet.Shared/Internal/InternalUtils.cs

```diff
@@ -495,6 +495,32 @@ internal static ListedResponse<T> AccessParameterReservedApiArray<T>(this Tokens
             list.Remove(kvp);
             return t.AccessApiArrayImpl<T>(m, uri.Replace(string.Format("{{{0}}}", reserved), kvp.Value.ToString()), list, "");
         }
+
+        internal static T AccessParameterReservedApi<T>(this TokensBase t, MethodType m, string uri, IEnumerable<string> reserveds, IEnumerable<KeyValuePair<string, object>> parameters)
```

Comment:
So you can remove the `string reserved` version of this (Ln. 481-497). Be sure to do it also for `-Array`, `-Async`, and `-ArrayAsync`, respectively.